Podfile

```diff
@@ -14,3 +14,8 @@ target 'ModernAVPlayer_Example' do
 
   end
 end
+
+target 'ModernAVPlayer_Example_tvOS' do
```

Comment:
I still have some issues when fetch your pr and build.

When executing pod install, I see some new files added and everything's fine. 
can you try again from your side a new pod install and let me know ?

<img width="720" alt="Screenshot 2020-11-18 at 12 59 37" src="https://user-images.githubusercontent.com/972638/99527994-ebf15c80-299d-11eb-9897-c67bc22b0a41.png">

Reply:
Included those changed into PR with a latest commit